packages/node/src/types.ts

```diff
@@ -372,6 +372,7 @@ export type NodeCloudProvider = 'local' | 'aws';
 
 export interface NodeSettings {
   readonly airnodeIdShort?: string;
+  readonly airnodeWalletMnemonic?: string;
```

Comment:
Shouldn't these be required?

Reply:
I don't actually think the mnemonic should be required because you can run the deployer without it and it will be created for you. Yes, you have to add it there afterwards but still.

And I don't really get the `airnodeIdShort` there. As far as I can tell it's never read from the `config.json`. It's not even in the new documentation. I think that should have been removed some time ago but was forgotten.

Reply:
I see your point. I still think it should be strictly defined in node. I would define a subtype in deployer `type DeployerNodeSettings = Partial<NodeSettings>` and use that there instead.

Regarding `airnodeIdShort` there is a discussion, but it still appears to be used? https://api3workspace.slack.com/archives/C027Y2FAV0S/p1629381176035600

Reply:
> I see your point. I still think it should be strictly defined in node. I would define a subtype in deployer type DeployerNodeSettings = Partial<NodeSettings> and use that there instead.

Yep, makes sense, will do that.

> Regarding airnodeIdShort there is a discussion, but it still appears to be used? https://api3workspace.slack.com/archives/C027Y2FAV0S/p1629381176035600

I'm not saying we should get rid of the concept of `airnodeIdShort`. I'm just saying that it's never read from the configuration. It's always derived from `airnodeId` which is derived from the mnemonic. And it will be the same with the `airnodeAddress` and its short version.